pkg/cmd/helm/build/build.go

```diff
@@ -126,14 +126,19 @@ func (o *Options) Run() error {
 			}
 
 			for i, dependency := range chartDef.Dependencies {
-				c := &cmdrunner.Command{
-					Dir:  chartDir,
-					Name: o.HelmBinary,
-					Args: []string{"repo", "add", strconv.Itoa(i), dependency.Repository},
-				}
-				_, err = o.CommandRunner(c)
-				if err != nil {
-					return errors.Wrapf(err, "failed to add repository")
+				fmt.Printf("Adding repository for dependency %s\n", dependency.Name)
```

Comment:
so that it's consistent with other parts it's probably best if we switch from using `fmt.Printf` to
```
log.Logger().Infof("Adding repository for dependency %s", dependency.Name)
```